site/docs/pages/wallet/wallet.mdx

```diff
@@ -476,6 +476,37 @@ export default OnchainProviders;
 
 ::::
 
+## Examples
+
+### With Sign In With Ethereum (SIWE)
+
+To use Sign In With Ethereum (SIWE) with OnchainKit, you can use the `onConnect` prop in the `<ConnectWallet />` component. This will trigger the SIWE prompt when the user connects their wallet.
```

Comment:
```suggestion
To use [Sign In With Ethereum (SIWE)](https://docs.login.xyz/general-information/siwe-overview) with OnchainKit, you can use the `onConnect` prop in the `<ConnectWallet />` component. This will trigger the SIWE prompt when the user connects their wallet.
```